jetty-servlets/src/main/java/org/eclipse/jetty/servlets/IncludeExcludeBasedFilter.java

```diff
@@ -122,10 +122,28 @@ protected boolean shouldFilter(HttpServletRequest http_request, HttpServletRespo
 
         String content_type = http_response.getContentType();
         LOG.debug("Content Type is: {}",content_type);
-        content_type = (content_type == null)?"":content_type;
-        String mime_type = MimeTypes.getContentTypeWithoutCharset(content_type);
 
-        LOG.debug("Mime Type is: {}",content_type);
+        String mime_type = "";
```

Comment:
Perhaps factor getting/guessing the mimetype out into a protected method so it can be overridden